schema/units.go

```diff
@@ -129,10 +129,12 @@ type Units interface {
 
 // NewUnits defines a new set of UnitsDefinition with the given parameters.
 func NewUnits(baseUnit *UnitDefinition, multipliers map[int64]*UnitDefinition) *UnitsDefinition {
-	return &UnitsDefinition{
+	ud := &UnitsDefinition{
 		BaseUnitValue:    baseUnit,
 		MultipliersValue: multipliers,
 	}
+	ud.updateReCache()
+	return ud
```

Comment:
If you intend to merge this change, you should also remove the lazy initialization code from `parse()` and anywhere else it is, since it will always find the cache already initialized.

Reply:
@jaredoconnell what do you think about this change?

Reply:
Are you asking about the call to update the cache?
To understand this, why are you doing it, and is it done elsewhere?

Reply:
I'm sorry. I forgot we don't need this change right now.